Base/Python/slicer/util.py

```diff
@@ -2395,6 +2395,140 @@ def createProgressDialog(parent=None, value=0, maximum=100, labelText="", window
       setattr(progressIndicator, key, value)
   return progressIndicator
 
+
+from contextlib import contextmanager
+
+@contextmanager
+def raisePythonShell(raise=True):
```

Comment:
`raise` may be misleading (users may think that it is related to raising exceptions). I think "display" would be simpler and more clear.